modules/installation-special-config-encrypt-disk-tang.adoc

```diff
@@ -122,6 +137,11 @@ spec:
         mode: 420
         overwrite: true
         path: /etc/clevis.json
+  kernelArguments:
+    - rd.neednet=1 <1>
 EOF
 ----
++
+<1> Required. Enables networking even when `netroot` utilities are not available so that the asymmetric keys can be obtained from the Tang server.
```

Comment:
My previous comment also applies to this call out.

Reply:
Updated.